sewer/client.py

```diff
@@ -122,6 +123,7 @@ def __init__(
         if not domain_alt_names:
             domain_alt_names = []
         self.domain_alt_names = domain_alt_names
+        self.domain_alt_names = list(set(self.domain_alt_names))
```

Comment:
@menduo I do not understand why you made this change

Reply:
This can remove duplicate domains, so the dns provider class will not send duplicate DNS record request.